torntools/scripts/content/global/ttGlobal.js

```diff
@@ -784,3 +787,17 @@ function addPeopleBoxFilter() {
 		};
 	}
 }
+
+function hideGymHighlight() {
+	if (settings.pages.gym.hide_gym_highlight) {
+		let navGym = doc.find("#nav-gym");
+		let navGymClass = navGym.className.substring(navGym.className.indexOf("available"));
```

Comment:
I'll accept this, but it's dangerous to assume that the class will always be the last class in the row.

Reply:
And that is why I have created _navGymClass_ .